tests/js/client/load-balancing/load-balancing-key-generator-noauth-cluster.js

```diff
@@ -90,18 +90,60 @@ function KeyGeneratorSuite () {
     return res;
   }
 
+  function generateCollectionDocs(name) {
+    // check that the generated keys are sequential when we send the requests
+    // via multiple coordinators.
+    db._create(name, {keyOptions: {type: "autoincrement"}});
+    assertNotEqual("", db[name].properties().distributeShardsLike);
+    let lastKey = null;
+    let url = "/_db/" + cn + "/_api/document/" + name;
+    // send documents to both coordinators
+    let i = 0;
+    let startingKey = null;
+    if (Number(name[name.length - 1]) === 1) {
+      startingKey = "abc123";
```

Comment:
The purpose of a key generator is that it generates the keys itself.
Using a key such as `abc123` somewhat defeats that purpose.

What is missing here in this test is the usage of an autoincrement key generator with different `offset` and/or `increment` values. These tests should be added.

Reply:
Will address